Registry/Registry.EM

```diff
@@ -4,6 +4,14 @@
 include registry.dimspec
 include registry.em_shared_collection
 
+# WRF-CMAQ coupled model
+rconfig  integer   wrf_cmaq_option      namelist,wrf_cmaq     1        0
+rconfig  integer   wrf_cmaq_freq        namelist,wrf_cmaq     1        1
+rconfig  integer   met_file_tstep       namelist,wrf_cmaq     1    10000
+
+rconfig  logical   direct_sw_feedback   namelist,wrf_cmaq     1  .false.
+rconfig  logical   feedback_restart     namelist,wrf_cmaq     1  .false.
```

Comment:
@davegill @dwongepa Should these line be added to Registry.EM_COMMON?

Reply:
@davegill Do you have a preference where these couple of new lines should be placed? It could be placed in the new registry file added: registry.WRF-CMAQ-twoway, could it?